src/main/webapp/WEB-INF/tags/instructor/feedbackEdit/newQuestionForm.tag

```diff
@@ -111,6 +111,7 @@
                             title="<%= Const.Tooltips.FEEDBACK_QUESTION_INPUT_DESCRIPTION %>"
                             tabindex="9">
                         </div>
+                        <input type="hidden" name="questiondescription">
```

Comment:
`questiondescription` should come from some constant in `Const.ParamsNames`, right?